_security/configuration/disable.md

```diff
@@ -43,22 +60,62 @@ If you disable the Security plugin in `opensearch.yml` (or delete the plugin ent
    COPY --chown=opensearch-dashboards:opensearch-dashboards opensearch_dashboards.yml /usr/share/opensearch-dashboards/config/
    ```
 
-   In this case, `opensearch_dashboards.yml` is a "vanilla" version of the file with no entries for the Security plugin. It might look like this:
+1. To build the new Docker image, run the following command:
+
+   ```bash
+   docker build --tag=opensearch-dashboards-no-security .
+   ```
+
+1. In `docker-compose.yml`, change `opensearchproject/opensearch-dashboards:{{site.opensearch_dashboards_version}}` to `opensearch-dashboards-no-security`.
+1. Change `OPENSEARCH_HOSTS` or `opensearch.hosts` to `http://` rather than `https://`.
+1. Enter `docker-compose up`.
+
+### Tarball 
+
+1. Stop the running instance of OpenSearch Dashboards (Ctrl+C).
```

Comment:
Where would the user stop their Dashboards instance using Ctrl + C? In their specific cluster? In their local instance?

Reply:
Updated it to:
Navigate to the `/bin` directory in your OpenSearch Dashboards installation folder and stop the running OpenSearch Dashboards instance by pressing `Ctrl + C`.

Reply:
@john-eliatra: That sounds ok to me.